stubs/D3DShot/d3dshot/capture_outputs/pil_capture_output.pyi

```diff
@@ -0,0 +1,24 @@
+from ctypes import _CVoidConstPLike
+from typing import TypeVar
+from typing_extensions import TypeAlias
+
+from d3dshot.capture_output import CaptureOutput
+from PIL import Image
+
+_Unused: TypeAlias = object
+_ImageT = TypeVar("_ImageT", bound=Image.Image)
+
+class PILCaptureOutput(CaptureOutput):
+    def __init__(self) -> None: ...
+    def process(
+        self,
+        pointer: _CVoidConstPLike,
+        pitch: int,
+        size: int,
+        width: int,
+        height: int,
+        region: tuple[int, int, int, int],
+        rotation: int,
+    ) -> Image.Image: ...
+    def to_pil(self, frame: _ImageT) -> _ImageT: ...
+    def stack(self, frames: _ImageT, stack_dimension: _Unused) -> _ImageT: ...  # type: ignore[override]
```

Comment:
Isn't `frames` supposed to be a `Sequence` here? (I believe you can also get rid of the `_Frames: TypeAlias = Image.Image | Sequence[Incomplete]` alias in the base class, and just use `Sequence[Incomplete]` as the annotation for `frames` in the base class. That should mean you can get rid of the `type: ignore` here.)

```suggestion
    def stack(self, frames: Sequence[_ImageT], stack_dimension: _Unused) -> Sequence[_ImageT]: ...
```

Reply:
Humm, might make more sense. The numpy and pytorch implementations return a single item. But those would themselves be a container (ndarray and Tensor)